### DISCUSSION TITLE
Action custom branding icon?

### DISCUSSION THREAD

**Discussion by aleks-ivanov** [community/community#26767] [Actions]
<p>How are these custom icons added to <code>branding:</code> ?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/5/75a8fbac1a2fd3eedd89089833e207b4aa340d07.png" data-download-href="/uploads/short-url/gMRUkRyTWDX2d7e3JdRwI9x0wU7.png?dl=1" title="Screenshot_1" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/74899441/181099813-ef311f7d-0cda-4b76-8582-4a83f28079c5.png" alt="Screenshot_1" data-base62-sha1="gMRUkRyTWDX2d7e3JdRwI9x0wU7" width="690" height="386" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/5/75a8fbac1a2fd3eedd89089833e207b4aa340d07_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot_1</span><span class="informations">833×466 45.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by dave-murray** [comment]
<p>Good question. As the author of multiple Actions I’d love to be able to have consistent branding across my actions rather than “well all the icons are the same colour.”</p>

**Comment by aleks-ivanov** [comment]
<p>I suspect that the author has to be an organization and somehow automatically is applied from the organization’s avatar or something, but I haven’t been able to confirm.</p>

**Comment by muzimuzhi** [comment, marked as answer]
<p>This <a href="https://github.community/t/can-one-set-a-custom-image-icon-for-a-github-action-or-github-workflow-or-only-github-apps/17513/4">reply</a> to a similar question “<a href="https://github.community/t/can-one-set-a-custom-image-icon-for-a-github-action-or-github-workflow-or-only-github-apps/17513">Can one set a custom image (icon) for a GitHub Action or GitHub Workflow? Or only GitHub Apps?</a>”, from a github staff, says</p>
<blockquote>
<p>Yes, companies we partner with are able to have custom icons. It’s setup by our team.</p>
</blockquote>

**Reply by kislerdm** [reply]
Plus one to the question. How to request a custom icon for the action?

**Comment by aleks-ivanov** [comment]
<p>Thanks!</p>
<p>Hmmm, that’s a weird thing to gate like that.</p>

**Comment by AlexAtkinson** [comment]
How can I partner with you to be able to customize the logo, @muzimuzhi?
*Shake* - we're partners. Let's get to it, ya? (Edit: I applied for realz...)
(I feel like I'm back in 2004...)
